code/game/objects/items/toys.dm

```diff
@@ -904,8 +909,248 @@
 		'sound/weapons/cablecuff.ogg' = 1,
 	)
 
+/obj/item/toy/plushie/skrellplushie
+	name = "skrell plushie"
+	desc = "The latest from 'SoftSkrells.net', features its very own headpocket! Warble!"
+	icon_state = "plushie_skrell"
+	COOLDOWN_DECLARE(warble_cooldown)
+	var/obj/item/headpocket_item
+
+/obj/item/toy/plushie/skrellplushie/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Alt-click to put something small inside the headpocket, or take an item out.</span>"
+
+/obj/item/toy/plushie/skrellplushie/AltClick(mob/user)
+	if(!Adjacent(user))
+		return
+	var/obj/item/I = user.get_active_hand()
+	if(I == src)
+		return
+	if(!I)
+		if(!headpocket_item)
+			return
+		to_chat(user, "<span class='notice'>You remove [headpocket_item] from [src].</span>")
+		headpocket_item.forceMove(get_turf(src))
+		user.put_in_hands(headpocket_item)
+		headpocket_item = null
+		return
+	if(I.w_class > WEIGHT_CLASS_SMALL)
+		to_chat(user, "<span class='warning'>You cannot fit [I] in [src]!</span>")
+		return
+	if(!iscarbon(user))
+		return
+	if(headpocket_item)
+		to_chat(user, "<span class='warning'>[src] already has an item in it's headpocket!</span>")
+		return
+	if(!user.drop_item())
+		to_chat(user, "<span class='warning'>You cannot slip [I] inside [src]!</span>")
+		return
+	user.visible_message("<span class='notice'>[user] places [I] into [src].</span>", "<span class='notice'>You place [I] into [src].</span>")
+	add_fingerprint(user)
+	I.forceMove(src)
+	headpocket_item = I
+
+/obj/item/toy/plushie/skrellplushie/Destroy()
+	QDEL_NULL(headpocket_item)
```

Comment:
i suggest make this drop the item instead

Reply:
Done